build.gradle

```diff
@@ -22,13 +22,12 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.cloud:spring-cloud-aws-messaging:2.2.6.RELEASE'
-
+    implementation 'org.springframework.boot:spring-boot-starter-oauth2-resource-server'
     implementation 'net.logstash.logback:logstash-logback-encoder:7.2'
-
     implementation 'com.github.ben-manes.caffeine:caffeine:3.1.1'
     implementation 'org.apache.commons:commons-csv:1.9.0'
-
     implementation 'org.flywaydb:flyway-core:8.5.13'
+    implementation 'org.projectlombok:lombok:1.18.22'
```

Comment:
does this really need to be implementation? I've never understood which ones lombok needs, but please use the lombok variable here anyway